cargo-culture/Cargo.toml

```diff
@@ -2,9 +2,15 @@
 name = "cargo-culture"
 version = "0.1.0"
 authors = ["Zachary Pierce <zpierce@polysync.io>"]
+description = "Tool to check Rust project conventions"
+license = "MIT"
+repository = "https://github.com/PolySync/cargo-culture"
+documentation = "https://docs.rs/cargo-culture"
+homepage = "https://github.com/PolySync/cargo-culture"
+categories = ["development-tools"]
 
 [dependencies]
-cargo-culture-kit = { path = "../cargo-culture-kit" }
+cargo-culture-kit = { path = "../cargo-culture-kit", version = "0.1.0" }
```

Comment:
Is this the normal convention for dealing with multiple projects in the same workspace? I could see it being a little odd when versions get bumped, or perhaps working strangely when imported from another crate. (this is mostly me learning, I don't have a specific complaint here)

Reply:
Yep, it's necessary to have a fixed version number for packaging purposes, even in the same workspace.

Other examples:
* [regex using regex-syntax](https://github.com/rust-lang/regex/blob/b5ef0ec281220d9047fed199ed48c29af9749570/Cargo.toml#L33)
* [rand using rand_core](https://github.com/rust-lang-nursery/rand/blob/b92c0e6b916984ced64e4a222d3e06b9ad4b841f/Cargo.toml#L32)